### PR TITLE
feat: Invoke the image upload and image recognition interface to real…

### PR DESCRIPTION
**Pull request by Fleurxxx**
…ize the function of carrying picture dialogue

English | [简体中文](https://github.com/opentiny/tiny-engine-webservice/blob/main/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine-webservice/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Introduced file upload functionality to the AI chat interface, allowing users to upload files for processing.
	- Enhanced capabilities of the AI chat service to process uploaded files and retrieve content from AI services.

- **Bug Fixes**
	- Improved error handling during file uploads to ensure better user experience and feedback.

### CODE REVIEW

File: app/service/app-center/aiChat.ts [hunk 1]
```diff
@@ -162,5 +162,47 @@ export default class AiChat extends Service {
...
+   */
+
+  async getFileContentFromAi(fileStream: any, chatConfig: any) {
+    const answer = await this.requestFileContentFromAi(fileStream, chatConfig);
+    return this.ctx.helper.getResponseData({
+      originalResponse: answer
+    });
+  }
+
+  async requestFileContentFromAi(file: any, chatConfig: any) {
```
Comment (chilingling) [inline app/service/app-center/aiChat.ts]: typescript 类型能否完善一下？

File: app/service/app-center/aiChat.ts [hunk 2]
```diff
@@ -162,5 +162,47 @@ export default class AiChat extends Service {
...
+    const { ctx } = this;
+    let res: any = null;
+    try {
+      // 文件上传
+      const aiUploadConfig = this.config.uploadFile(file, chatConfig.token);
+      const { httpRequestUrl, httpRequestOption } = aiUploadConfig[chatConfig.model];
+      this.ctx.logger.debug(httpRequestOption);
+      res = await ctx.curl(httpRequestUrl, httpRequestOption);
+      const imageObject = JSON.parse(res.res.data.toString());
+      const fileObject = imageObject.data[0].id;
```
Comment (chilingling) [inline app/service/app-center/aiChat.ts:195]: 这块解析图片的约定感觉可以看看其他家是不是也是类似的，不然后续适配其他家的图片上传就不适用了

Comment (Fleurxxx) [reply app/service/app-center/aiChat.ts:195]: kimi的图片解析这里是上传到moonshot的云端上再进行解析，的确实会有不适用其他AI问题，当时可能没考虑到，但是我预留了模型类型的字段可以后期做特判，现在暂定是想先跑通kimi的，再去考虑其他的AI中图片上传的统一适配问题。

Comment (chilingling) [reply app/service/app-center/aiChat.ts:195]: ok

**Comment by Fleurxxx** [comment]
@chilingling 再帮我看一下好嘛

**Review by chilingling — APPROVED** [review]

**Pull request merged**